R/derive_interp_records.R

```diff
@@ -40,16 +47,13 @@
 #'     AGE = round(AGE * 30.4375)
 #'   ) %>%
 #'   # Interpolate the AGE by SEX
-#'   group_by(SEX) %>%
 #'   # Ensure first that Age unit is "DAYS"
-#'   do(derive_interp_records(., parameter = "HEIGHT")) %>%
-#'   ungroup() %>%
-#'   # Keep patients >= 2 yrs only - Remove duplicates for 730 Days old which
-#'   # must come from WHO metadata only
-#'   filter(AGE >= 730.5)
+#'   derive_interp_records(., parameter = "HEIGHT")
```

Comment:
```suggestion
#'   derive_interp_records(., by_vars = exprs(SEX), parameter = "HEIGHT")
```
as `by_vars` should be NULL as default

Reply:
Implemented